clusterman/mesos/mesos_cluster_connector.py

```diff
@@ -73,6 +73,9 @@ def get_resource_allocation(self, resource_name: str) -> float:
     def get_resource_total(self, resource_name: str) -> float:
         return sum(getattr(total_agent_resources(agent), resource_name) for agent in self._agents_by_ip.values())
 
+    def freeze_agent(self, agent_id: str) -> None:
+        return
```

Comment:
https://github.com/Yelp/clusterman/pull/164#discussion_r914209829

Reply:
I meant that we could add a comment here mentioning why we're not doing the equivalent action in mesos (which would be putting the agent in maintenance mode)

although, actually: it might be good to add a log statement or something here - otherwise, the logs for mesos autoscalers will incorrectly say `"freezing {instance_id} for termination"` if i'm reading things correctly

Reply:
Hmm. I got your point. Should I implement same logic for mesos? I skipped it because I thought that we don't use mesos anymor

Reply:
we still use mesos in a few cases (although these are going away slowly)

it's not worth adding the same logic here (imo), but we should probably log something so that the logs aren't confusing. right now they'll say that mesos instances are being frozen, but that's not actually true

worst case, a comment that reads something like `# intentionally not implemented as Mesos is deprecated at Yelp` would work just as well (but would require whoever gets confused by the misleading log message in a mesos autoscaler to read the code :p)